crates/nu-command/src/conversions/into/float.rs

```diff
@@ -0,0 +1,181 @@
+use nu_cmd_base::input_handler::{operate, CellPathOnlyArgs};
+use nu_engine::CallExt;
+use nu_protocol::{
+    ast::{Call, CellPath},
+    engine::{Command, EngineState, Stack},
+    Category, Example, PipelineData, Record, ShellError, Signature, Span, SyntaxShape, Type, Value,
+};
+
+#[derive(Clone)]
+pub struct SubCommand;
+
+impl Command for SubCommand {
+    fn name(&self) -> &str {
+        "into float"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("into float")
+            .input_output_types(vec![
+                (Type::Int, Type::Float),
+                (Type::String, Type::Float),
+                (Type::Bool, Type::Float),
+                (Type::Float, Type::Float),
+                (Type::Table(vec![]), Type::Table(vec![])),
+                (Type::Record(vec![]), Type::Record(vec![])),
+                (
+                    Type::List(Box::new(Type::Any)),
+                    Type::List(Box::new(Type::Float)),
+                ),
+            ])
+            .rest(
+                "rest",
+                SyntaxShape::CellPath,
+                "for a data structure input, convert data at the given cell paths",
+            )
+            .allow_variants_without_examples(true)
+            .category(Category::Conversions)
+    }
+
+    fn usage(&self) -> &str {
+        "Convert data into floating point number."
+    }
+
+    fn search_terms(&self) -> Vec<&str> {
+        vec!["convert", "number", "floating"]
```

Comment:
```suggestion
        vec!["convert", "number", "floating", "decimal"]
```

Reply:
I approved the PR without it so I could go either way. I just thought for non-technical people, it may be helpful since this is only used in search terms. I'm fine with whatever you decide.

Reply:
Ship it